### DISCUSSION TITLE
implementation in Django

### DISCUSSION THREAD

**Discussion by sierraa117** [miguelgrinberg/python-socketio#1279] [Q&A]
I want to implement socket-io on django can anyone help me with this??

**Comment by miguelgrinberg** [comment]
There is a Django example in this repository. Have you seen it?

**Comment by sierraa117** [comment]
yes, I saw that example but unable to run it on my pc can you provide me the basic instruction .
Not Found: /socket.io/
INFO:     127.0.0.1:51225 - "GET /socket.io/?EIO=4&transport=polling&t=OliHh5m HTTP/1.1" 404 Not Found
 I keep getting this error

**Reply by miguelgrinberg** [reply]
I'm not sure why. To me it looks like you are running a different app, not the example that I have on this repository.

The instructions to run the app are in the README. See https://github.com/miguelgrinberg/python-socketio/tree/main/examples/server/wsgi/django_socketio.

**Comment by sierraa117** [comment]
@miguelgrinberg Thank you so much for your help. I'm able to run the socket but I have to use to integrate web socket  IVR and IVR provider is asking for events on which my response function is calling I think its not possible using this and Django channels do not have events

**Reply by miguelgrinberg** [reply]
This package implements Socket.IO, not WebSocket. If your provider uses WebSocket, then this package is not going to work.

**Comment by sierraa117** [comment]
yes i want to implement socket.IO but I dont want to integrate it with front end I want to integrate it with IVR and there is no link for socket.IO from which I can test it using postman
